datarepo-client/build.gradle

```diff
@@ -5,8 +5,8 @@ plugins {
     id 'org.hidetake.swagger.generator'
 }
 
-sourceCompatibility = 11
-targetCompatibility = 11
+sourceCompatibility = 17
```

Comment:
I think this should stay at 11

Reply:
Awesome - Done and confirmed that the jar generated on client build is Java version 11.